### ISSUE TITLE
StaphBase uses taxon references for name authorships

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#685]
For higher taxa in StaphBase the authorship is at least in some cases not the nomenclatural author, but the taxonomic concept authorship. For example Staphylinidae Latreille, 1802 has these 2 synonyms:

[= Staphylinidae ICZN, 1959](https://www.checklistbank.org/dataset/299029/nameusage/9J9HP) [[2]](https://www.checklistbank.org/dataset/299029/taxon/9J9HN#col-refererence-14e222e1-812d-4d97-8b60-8f03ec3a21d0)
[= Staphylinoidea Ganglbauer, 1895](https://www.checklistbank.org/dataset/299029/nameusage/B6LT8) [[3]](https://www.checklistbank.org/dataset/299029/taxon/9J9HN#col-refererence-b7c3acc5-e51a-4fce-8e6b-4b8aaaaadba4)

`ICZN` is not an author and [Ganglbauer](https://www.biodiversitylibrary.org/item/36611#page/7/mode/1up) also is probably not the nomenclatural author of the family name Staphylinoidea. Instead they should be concept authors, i.e. the accordingTo reference. 

<img width="1228" alt="image" src="https://github.com/user-attachments/assets/90ed2222-f483-4c51-aa58-2a472779923d">

https://www.checklistbank.org/dataset/299029/taxon/9J9HN

**Comment by DaveNicolson** [comment]
Yeah, those look like nomenclaturally significant usages... Here's what Bouchard et al. (2011) indicated about the nomenclatural history of the family-group name based on Staphylinus:

"Family. Staphylinidae
Latreille, 1802

Staphyliniae Latreille, 1802: 124 [stem: Staphylin-]. Type genus: Staphylinus Linnaeus, 1758 [placed on the Official List of Generic Names in Zoology (ICZN 1959a)]. Comment: placed on the Official List of Family-Group Names in Zoology (as Staphylinidae Latreille, [1803-1804]) and “Staphylinii Latreille, [1803-1804]” placed on the Official Index of Rejected and Invalid Family-Group Names in Zoology (ICZN 1959a); First Reviser (Staphylinidae Latreille, 1802 vs Pselaphidae Latreille, 1802) not determined, current usage maintained."

Per the Principle of Coordination, Staphylin- names at any of the family-group ranks based on Staphylinus should be attributed to Latreille, 1802.

**Comment by yroskov** [comment]
@gdower, could we have look on this, please?

**Comment by gdower** [comment]
That's what is provided in the data, so it would probably be best to get it corrected in the source dataset. We could notify StaphBase, although there are also plans in the near future to move data curation into TaxonWorks with several people having expressed interest in curating the data in TaxonWorks. That will likely help get issues like this fixed.
